editor/components/block-toolbar/index.js

```diff
@@ -11,26 +11,28 @@ import BlockSwitcher from '../block-switcher';
 import BlockControls from '../block-controls';
 import BlockFormatControls from '../block-format-controls';
 
-function BlockToolbar( { block, mode } ) {
-	if ( ! block || ! block.isValid || mode !== 'visual' ) {
+function BlockToolbar( { block, mode, selectedBlockUIDs } ) {
+	if ( selectedBlockUIDs.length < 2 && ( ! block || ! block.isValid || 'visual' !== mode ) ) {
```

Comment:
- Should we also render nothing if the `selectedBlockUIDs` is empty?
- Instead of passing the whole `block` object. It seems we can just get the `isValid` flag instead as a prop that way this component will only rerender if the flag changes and not other block properties (performance tip)